sdk/storage/azblob/blockblob/client.go

```diff
@@ -165,6 +165,20 @@ func (bb *Client) Upload(ctx context.Context, body io.ReadSeekCloser, options *U
 	return resp, err
 }
 
+// PutBlobFromURL - The Put Blob from URL operation creates a new Block Blob where the contents of the blob are read from
+// a given URL. Partial updates are not supported with Put Blob from URL; the content of an existing blob is overwritten
+// with the content of the new blob. To perform partial updates to a block blob’s contents using a source URL, use the Put
+// Block from URL API in conjunction with Put Block List.
+// For more information, see https://learn.microsoft.com/rest/api/storageservices/put-blob-from-url
+func (bb *Client) PutBlobFromURL(ctx context.Context, contentLength int64, copySource string, options *PutBlobFromURLOptions) (PutBlobFromURLResponse, error) {
```

Comment:
The [documentation](https://learn.microsoft.com/en-us/rest/api/storageservices/put-blob-from-url#request-headers) states the following for the `Content-Length` header.
> <html>
<body>
<!--StartFragment-->

Content-Length | Required. Specifies the number of bytes being transmitted in the request body. The value of this header must be set to 0. When the length isn't 0, the operation fails with the status code 400 (Bad Request).
-- | --

<!--EndFragment-->
</body>
</html>

If this is the case, we should not have this as a required parameter and always pass its value as 0 to the generated code.

Reply:
Thank you for pointing this out! Fixed now!